ocamldoc/odoc_print.ml

```diff
@@ -14,6 +14,7 @@
 (**************************************************************************)
 
 open Format
+;; Printtyp.Naming_context.enable false
```

Comment:
Unusual style in the compiler: the standard style is `let () = ...`, which ensures the absence of partial application.

Reply:
Fixed, thanks.